imports/node-app/core-services/shop/mutations/updateShop.js

```diff
@@ -2,7 +2,6 @@ import SimpleSchema from "simpl-schema";
 import ReactionError from "@reactioncommerce/reaction-error";
 import { ShopLogoUrls, StorefrontUrls } from "/imports/collections/schemas";
```

Comment:
@aldeed I think I might need to do something here with these schemas.

Do all the schemas in the `shops` file need to be moved into this package? Or just the two imported here?

https://github.com/reactioncommerce/reaction/blob/refactor-kieckhafer-movePackageToNodeApp-shop/imports/collections/schemas/shops.js#L207

Reply:
Seems best to move all. I did it for you after merging in develop in https://github.com/reactioncommerce/reaction/pull/5635/commits/80633e67d8a1744a1acbf3627c62626ccce014f1